Assets/MRTK/Providers/Windows/WindowsDictationInputProvider.cs

```diff
@@ -86,7 +86,9 @@ public async Task StartRecordingAsync(GameObject listener = null, float initialS
             using (StartRecordingAsyncPerfMarker.Auto())
             {
 #if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
-                if (IsListening || isTransitioning || Service == null || !Application.isPlaying)
+                IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
```

Comment:
Is this cast needed? I think `Service` is already of type `IMixedRealityInputSystem`, due to the way `BaseInputDeviceManager` is set up (there are several of these additions in this file).

Reply:
I have the same question... But at the same time I can see how this line can help newcomers better understand what "service" is (although arguably VS can do the same job)

Reply:
I was wondering the same thing actually! I had pulled this code in from a different repo, and am currently asking the team regarding why they made the change. If we don't get an answer by end of day we can just remove the cast.

Reply:
Based on the conversation between Julia, the team and me, the redundant casts are removed.